content/applications/services/helpdesk/overview/ratings.rst

```diff
@@ -68,13 +70,15 @@ team has received over the last thirty days. Specific written feedback will not
 statistics of the team's performance will be visible.
 
 .. important::
-   In order to display ratings on the customer portal, a team has to have their visibility setting
-   set to :guilabel:`Invited portal users and all internal users`. This setting is found on the
-   team's settings page under :guilabel:`Visibility`.
+   To display ratings on the customer portal, a team **must** have their visibility setting set to
+   :guilabel:`Invited portal users and all internal users`. To enable this setting, navigate to
+   :menuselection:`Helpdesk app --> Configuration --> Helpdesk Teams`. Select a team from the list
+   and navigate to the settings page. Scroll to the :guilabel:`Visibility & Assignment` section, and
+   tick the :guilabel:`Invited portal users and all internal users` checkbox.
```

Comment:
```suggestion
   tick the :guilabel:`Invited portal users and all internal users` checkbox.
```

this isn't the exact thing on the UI

Should be: Invited portal users and all internal users (public)